### ISSUE TITLE
[Unhandled promise rejection: TypeError: _reactNative.NativeModules.RNDatePickerAndroid.dismiss is not a function.

### ISSUE THREAD

**Issue by pira998** [mmazzarolo/react-native-modal-datetime-picker#520]
In Android, it shows yellow box warning after picking a date.

[Unhandled promise rejection: TypeError: _reactNative.NativeModules.RNDatePickerAndroid.dismiss is not a function. (In '_reactNative.NativeModules.RNDatePickerAndroid.dismiss()', '_reactNative.NativeModules.RNDatePickerAndroid.dismiss' is undefined)]
* http://127.0.0.1:19001/node_modules%5Cexpo%5CAppEntry.bundle?platform=android&dev=true&hot=false&minify=false:134246:104 in dismiss$
- node_modules\regenerator-runtime\runtime.js:63:36 in tryCatch
- node_modules\regenerator-runtime\runtime.js:293:29 in invoke
- node_modules\regenerator-runtime\runtime.js:63:36 in tryCatch
- node_modules\regenerator-runtime\runtime.js:154:27 in invoke
- node_modules\regenerator-runtime\runtime.js:189:16 in PromiseImpl$argument_0
- node_modules\react-native\node_modules\promise\setimmediate\core.js:45:6 in tryCallTwo
- node_modules\react-native\node_modules\promise\setimmediate\core.js:200:22 in doResolve
- node_modules\react-native\node_modules\promise\setimmediate\core.js:66:11 in Promise
- node_modules\regenerator-runtime\runtime.js:188:15 in callInvokeWithMethodAndArg
- node_modules\regenerator-runtime\runtime.js:211:38 in enqueue
- node_modules\regenerator-runtime\runtime.js:238:8 in exports.async

**Comment by appseugene** [comment]
What was the solution on this one?

**Comment by pira998** [comment]
I am still getting that same warning ⚠️

**Comment by mariojgmaster** [comment]
I got that warning too. I tried to use "DateTimePickerModal" from "react-native-modal-datetime-picker", but nothing has changed.

**Comment by mariojgmaster** [comment]
Hi, Guys!
I finally get it.

To make warnings stop, I followed this steps:

1) I opened node_modules > @react-native-community > datetimepicker > src > datetimepicker.android.js

2) I commented the onChange method in this following code snippet (inside switch-case):
    case DISMISS_ACTION:
        default:
           event.type = 'dismissed';
           onChange(event);
           break;

3) I saved it.
4) I ran the component on device.
5) I turned the method back.
6) I uncommented the method again.
7) I Repeated the topic 4.

It worked for me. I hope it can help you too.
